cluster/src/main/java/com/hartwig/pipeline/calling/somatic/SageV2CallerOutput.java

```diff
@@ -0,0 +1,29 @@
+package com.hartwig.pipeline.calling.somatic;
+
+import java.util.Optional;
+
+import com.hartwig.pipeline.StageOutput;
+import com.hartwig.pipeline.execution.PipelineStatus;
+import com.hartwig.pipeline.storage.GoogleStorageLocation;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface SageV2CallerOutput extends StageOutput {
+
+    default String name() {
+        return SageV2Caller.NAMESPACE;
+    }
+
+    PipelineStatus status();
+
+    Optional<GoogleStorageLocation> maybeFinalSomaticVcf();
```

Comment:
As mentioned before I think best we remove this until there is a downstream dependency, might confuse someone.